extensions/amp-list/0.1/test/test-amp-list-load-more.js

```diff
@@ -145,5 +150,61 @@ describes.realWin('amp-list component', {
       await list.layoutCallback();
       expect(attemptChangeHeightSpy).to.be.calledOnceWith(50);
     });
+
+  });
+
+  describe('loading states', () => {
+
+    beforeEach(() => {
+      expect(isExperimentOn(win, 'amp-list-load-more')).to.be.true;
+
+      element = doc.createElement('amp-list');
+      list = new AmpList(element);
+
+      sandbox.stub(list, 'getAmpDoc').returns(ampdoc);
+      sandbox.stub(list, 'getFallback').returns(null);
+
+      sandbox.stub(list, 'mutateElement').callsFake(mutator => {
+        mutator();
```

Comment:
I think we should centralize these fakes somewhere, because the behavior here is different than the runtime. For example, if you have:

```js
let foo = 5;
this.mutateElement(() => {
  console.log(foo);
});
foo = 6;
```

In the test, this will log `5`, but with the runtime, it will always log `6`. We should do the same as the `amp-pan-zoom` test:

https://github.com/ampproject/amphtml/blob/64696e75c6bb6685d72c754471440092cca65d8b/extensions/amp-pan-zoom/0.1/test/test-amp-pan-zoom.js#L35-L37

https://github.com/ampproject/amphtml/blob/64696e75c6bb6685d72c754471440092cca65d8b/extensions/amp-pan-zoom/0.1/test/test-amp-pan-zoom.js#L65-L68

Reply:
Added a new function in `test-helper.js` that's the same as `<amp-pan-zoom>`'s stubs. I'll create a separate PR for refactoring that to keep the scope of each PR clear and concise.